src/Utilities/Compiler/Extensions/ISymbolExtensions.cs

```diff
@@ -713,7 +713,7 @@ public static IEnumerable<AttributeData> GetAttributes(this ISymbol symbol, INam
                 return Enumerable.Empty<AttributeData>();
             }
 
-            return symbol.GetAttributes().Where(attr => attr.AttributeClass.Equals(attributeType));
+            return symbol.GetAttributes().Where(attr => SymbolEqualityComparer.Default.Equals(attr.AttributeClass, attributeType));
```

Comment:
You can use HasAttribute extension method here instead.

Reply:
Just had a look and I can't make this replacement here.